addons/website_sale/controllers/main.py

```diff
@@ -742,7 +751,7 @@ def address(self, **kw):
                             (not order.only_services and (mode[0] == 'edit' and '/shop/checkout' or '/shop/address'))
                     # We need to update the pricelist(by the one selected by the customer), because onchange_partner reset it
                     # We only need to update the pricelist when it is not redirected to /confirm_order
-                    if kw.get('callback', '') != '/shop/confirm_order':
+                    if kw.get('callback', False) != '/shop/confirm_order':
```

Comment:
What does this fix ?

Reply:
It gives consistency for how we are representing an undefined value. It can be left out of this fix, but I believe this can lead to confusion in the future.

Reply:
It could be removed IMHO, as it doesn't really add anything (and might give you conflicts in the forward-ports).

Reply:
I'm ok with dealing with the forward-ports. The original behavior that the rest of this PR changes wasn't wrong as such, it's a matter of which convention gets used throughout Odoo (and to fix an issue that results from different conventions being mixed). Functionally it doesn't matter if this little if test checks on `False` or `''`, but convention-wise it does, so that's why I prefer to include it.